benchmarks/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:latest
+
+RUN apt update && apt install -y wget git
+RUN apt update && apt install -y build-essential
+RUN apt update && apt install -y unzip
+RUN apt update && apt install -y curl
```

Comment:
Some notes about this:

- Only one update is necessary
- Group all dependencies into a single command, to prevent creating multiple layers
- Clean caches after installing, to reduce the image size
- Use apt-get instead of apt, it guarantees a stable command-line interface

My suggestion:

```
RUN apt-get update \
        && apt-get install -y wget git build-essential unzip curl \
        && rm -rf /var/lib/apt/lists/*
```

(Of course during development of the Dockerfile copy-paste of these lines may make sense, to re-use the cache.)